_shows/06_07/zoo_story.md

```diff
@@ -38,7 +38,7 @@ cast:
   - role: Jerry
     name: Magnus McCullagh
   - role: Peter
-    name: Theo Taptiklis
```

Comment:
So, a question for here, do we use the name they have in the industry, or the name they used here????